intelmq/bots/outputs/bro_file/output.py

```diff
@@ -0,0 +1,146 @@
+# -*- coding: utf-8 -*-
+"""
+Bro file output
+
+SPDX-FileCopyrightText: 2021 Marius Karotkis <marius.karotkis@gmail.com>
+SPDX-License-Identifier: AGPL-3.0-or-later
+"""
+
+import datetime
+import os
+from collections import defaultdict
+from pathlib import Path
+
+from intelmq.lib.bot import OutputBot
+
+BRO_INDICATOR_MAP = {
+    "source.ip": "Intel::ADDR",
+    "source.fqdn": "Intel::DOMAIN",
+    "source.url": "Intel::URL"
+}
+
+BRO_HEADER = "#fields\tindicator\tindicator_type\tmeta.desc\tmeta.cif_confidence\tmeta.source\n"
+
+
+class BroFileOutputBot(OutputBot):
+    _file = None
+    encoding_errors_mode = 'strict'
+    file: str = "/opt/intelmq/var/lib/bots/file-output/bro"
+    format_filename: bool = False
+    hierarchical_output: bool = False
+    keep_raw_field: bool = False
+    message_jsondict_as_string: bool = False
+    message_with_type: bool = False
+    single_key: bool = False
+    is_multithreadable = False
+
+    def init(self):
+        # needs to be done here, because in process() FileNotFoundError handling we call init(),
+        # otherwise the file would not be opened again
+        self._file = None
+
+        self.logger.debug("Opening %r file.", self.file)
+        self.format_filename = getattr(self, 'format_filename', False)
+        self.errors = getattr(self, 'encoding_errors_mode', 'strict')
```

Comment:
```suggestion
```
This is set to strict already on line 27, and it's ensured that it's always set. (if you remove it, you'd need to rename the other usages of this variable as well.